seedlet/package-dotorg.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/zsh
+find assets/sass/*.scss -type f -exec sed -i '' 's/-wpcom//g' {} \; 
+find assets/sass/*.scss -type f -exec sed -i '' 's/auto-loading-homepage, //g' {} \; 
+npm run build;
+mkdir seedlet;
+rsync -avz --exclude-from 'dotorg-exclude.txt' ./ seedlet
+find seedlet -type f -name '*.map' -delete # for some reason rsync won't exclude map files
+zip -r -X seedlet.zip seedlet
+rm -rf seedlet
```

Comment:
It might also be worth adding this line at the end: 
```suggestion
rm -rf seedlet
git restore ./
```